src/platforms/dotnet/guides/aspnet/performance/included-instrumentation.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: Included Instrumentation
+sidebar_order: 20
+description: "Learn what transactions are captured after tracing is enabled."
+---
+
+<Note>
+
+Capturing transactions requires that you first <PlatformLink to="/performance/">set up performance monitoring</PlatformLink> if you haven't already. You'll also need to enable the <PlatformLink to="/">base Sentry integration</PlatformLink>.
+
+</Note>
+
+ASP.NET integration, once enabled, captures each incoming HTTP request and turns it into a transaction with the help of a custom middleware. To enable tracing on ASP.NET, you need to update your `Global.asax.cs` file like this:
+
+```csharp
+using System;
+using System.Configuration;
+using System.Web.Mvc;
+using System.Web.Routing;
+using Sentry;
+using Sentry.AspNet;
+
+namespace AspNetMvc
+{
+    public class MvcApplication : System.Web.HttpApplication
+    {
+        private IDisposable _sentry;
+
+        protected void Application_Start()
+        {
+            // Set up the sentry SDK
+            _sentry = SentrySdk.Init(o =>
+            {
+                // We store the DSN inside Web.config; make sure to use your own DSN!
+                o.Dsn = ConfigurationManager.AppSettings["SentryDsn"];
```

Comment:
The user is free to move this to web.config but lets render the actual DSN here instead:

```suggestion
                o.Dsn = ___PUBLIC_DSN___
```